.pre-commit-config.yaml

```diff
@@ -60,3 +60,13 @@ repos:
     rev: 0.6.1
     hooks:
       - id: nbstripout
+  - repo: https://github.com/executablebooks/mdformat
+    rev: 0.7.16
+    hooks:
+    - id: mdformat
+      exclude: ^(warehouse/analyses/|warehouse/macros/|warehouse/models/|warehouse/secrets/|warehouse/seeds/|warehouse/snapshots/|warehouse/tests/)
```

Comment:
NOTE: this specific exclude pattern was put in place so that `warehouse/README.md` would continue to be linted

Reply:
a negative lookahead could be more straightfoward `^warehouse/(?!README.md)` and would also cover future folders

Reply:
Resolved in [06c9957](https://github.com/cal-itp/data-infra/pull/2906/commits/06c9957cd4d123473f8cd621857a5dd524c9dfed) and tested locally by reverting the linting changes from the previous commit and running pre-commit to see which files the new exclusions acted on.